andela/hyuga-cfh
# #133989809 Fix Linter

#### What does this PR do?
Ensures proper styling and indentation.
#### Description of Task to be completed?
Linting service that reports appropriate errors in files.
#### How should this be manually tested?
- Run `npm install` to ensure all the `ESLint` dependencies are installed and then;
- Run `npm run lint "filePath"` to run the lint on a specific file or folder and report all appropriate errors found. If you have `ESLint` installed globally on your machine, simply run `eslint "filePath"` to target a specific file/folder.

#### Any background context you want to provide?
`Indent` and `End-of-file` configurations are deprecated in `JSHint`. Hence the need to use `ESLint` in order to ensure such errors are reported.
#### What are the relevant pivotal tracker stories?
#133989809 Fix Linter
#### Screenshots (if appropriate)
NA
#### Questions:
NA
